src/ResourceManager/LogicApp/Commands.LogicApp/Cmdlets/IntegrationAccount/GetAzureIntegrationAccountAgreementCommand.cs

```diff
@@ -21,22 +21,32 @@ namespace Microsoft.Azure.Commands.LogicApp.Cmdlets
     /// <summary>
     /// Gets the integration account agreement by name 
     /// </summary>
-    [Cmdlet(VerbsCommon.Get, "AzureRmIntegrationAccountAgreement"), OutputType(typeof (object))]
+    [Cmdlet(VerbsCommon.Get, "AzureRmIntegrationAccountAgreement")]
+    [OutputType(typeof(object))]
```

Comment:
This change resulted in a breaking change build error. I think that as object return type provided no value this breaking change is unlikely to actually break any client. Is there a way to acknowledgement the breaking change as acceptable?

00:48:41   "D:\workspace\powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.LogicApp\Microsoft.Azure.Commands.LogicApp.dll","Microsoft.Azure.Commands.LogicApp.Cmdlets.GetAzureIntegrationAccountAgreementCommand","Get-AzureRmIntegrationAccountAgreement","0","1020","The cmdlet 'Get-AzureRmIntegrationAccountAgreement' no longer has output type 'Object'.","Make cmdlet 'Get-AzureRmIntegrationAccountAgreement' return type 'Object'."